### DISCUSSION TITLE
Web Interface

### DISCUSSION THREAD

**Discussion by Forts117** [chuot/rdio-scanner#411] [Q&A]
Hi all,

I've been using the web interface due to the mobile app issues. Question - how do I log out via the web to switch to a different login? I logged in to an individual user account to make sure it was working, now I can't find how to logout so I can switch back to my personal account. Surely I'm missing something simple here...

**Comment by rdmcvicar** [comment]
I love the software, but am not aware of a built-in means to do this.

The (iOS, and I assume Android) app allows multiple URLs and unlock codes to be stored.

For browsers, my workarounds include:
- opening private/incognito windows to demand an unlock code but not cache it. This is what I do to test other unlocks like you mention.
- opening different browsers and/or browser profiles to treat entered unlock codes separately.
- as a last resort, clearing the browser cache, forcing another login
- perhaps even more of a last resort (since I run my own server), resetting things on that end and demanding new unlocks from everyone

Happy rdio-scanning,

Rich McVicar
rdmcvicar tds.net
Middleton, WI

(Fellow rdio-scanners, feel free to reach out directly -- I enjoy trading sites / systems.)

**Comment by Forts117** [comment]
Oh... good idea on the incognito tab, I hadn't thought of that. I'll give that a try next time.
